package.json

```diff
@@ -22,10 +22,11 @@
     }
   ],
   "engines": {
-    "node": ">=0.12.0"
+    "node": ">=4"
   },
   "scripts": {
     "test": "xo && npm run prepublish && nyc ava",
+    "flow": "flow; test $? -eq 0 -o $? -eq 2",
```

Comment:
I don't understand what you're looking for. You want flow to never return an error exit code ?